### PR TITLE
Update Guild features list

### PR DESCRIPTION
**Pull request by Soheab**
## Summary

~~This updates the guild features list. Discord keeps changing them and not documenting them so I replaced everything with the API docs [here](https://discord.com/developers/docs/resources/guild#guild-object-guild-features) and the following are added by me:
`ANIMATED_BANNER, ENABLED_DISCOVERABLE_BEFORE, MEMBER_PROFILES, NEW_THREAD_PERMISSIONS, THREADS_ENABLED` from discord.py's features list.~~ 

This PR removes the hardcoded guild features and replaces them with a link to the docs instead, this way we don't have to look out for changes etc...

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/types/guild.py [hunk 1]
```diff
@@ -75,27 +75,32 @@ class _GuildOptional(TypedDict, total=False):
...
     "PRIVATE_THREADS",
+    "ROLE_ICONS",
     "SEVEN_DAY_THREAD_ARCHIVE",
     "THREE_DAY_THREAD_ARCHIVE",
+    "THREADS_ENABLED",
     "TICKETED_EVENTS_ENABLED",
     "VANITY_URL",
     "VERIFIED",
-    "VIP_REGIONS",
+    "VIP_REGION",
```
Comment (ow0x) [inline discord/types/guild.py:103]: I think `VIP_REGIONS` here should be kept as is as per [Discord API docs](https://discord.com/developers/docs/resources/guild#guild-object-guild-features) for guild object guild features.

Comment (Soheab) [reply discord/types/guild.py]: Fixed in [`3936f62`](https://github.com/iDevision/enhanced-discord.py/pull/152/commits/3936f6255c7bb3dafa4e604b5e9f01527a4ca86b), thanks!

**Comment by GnomedDev** [comment]
Why are all of these features documented here? Can't we just link to discord's docs for this since it's a straight copy paste?

**Comment by Soheab** [comment]
> Why are all of these features documented here? Can't we just link to discord's docs for this since it's a straight copy paste?

that would be nice, discord.py people were against it.

**Pull request merged**